server/src/test/java/io/unitycatalog/server/base/volume/BaseVolumeCRUDTest.java

```diff
@@ -149,38 +140,32 @@ public void testVolumeCRUD() throws ApiException {
     }
 
     VolumeInfo managedVolumeInfo = volumeOperations.getVolume(VOLUME_FULL_NAME);
-    assertEquals(VolumeType.MANAGED, managedVolumeInfo.getVolumeType());
-    assertEquals(
-        FileUtils.convertRelativePathToURI("/tmp/managed_volume"),
-        managedVolumeInfo.getStorageLocation());
-    assertEquals(VOLUME_FULL_NAME, managedVolumeInfo.getFullName());
-    assertEquals(VOLUME_NAME, managedVolumeInfo.getName());
-    assertEquals(CATALOG_NAME, managedVolumeInfo.getCatalogName());
-    assertEquals(SCHEMA_NAME, managedVolumeInfo.getSchemaName());
-    assertNotNull(managedVolumeInfo.getCreatedAt());
-    assertNotNull(managedVolumeInfo.getUpdatedAt());
+    assertThat(managedVolumeInfo.getVolumeType()).isEqualTo(VolumeType.MANAGED);
+    assertThat(managedVolumeInfo.getStorageLocation())
+        .isEqualTo(FileUtils.convertRelativePathToURI("/tmp/managed_volume"));
+    assertThat(managedVolumeInfo.getFullName()).isEqualTo(VOLUME_FULL_NAME);
+    assertThat(managedVolumeInfo.getName()).isEqualTo(VOLUME_NAME);
+    assertThat(managedVolumeInfo.getCatalogName()).isEqualTo(CATALOG_NAME);
+    assertThat(managedVolumeInfo.getSchemaName()).isEqualTo(SCHEMA_NAME);
+    assertThat(managedVolumeInfo.getCreatedAt()).isNotNull();
+    assertThat(managedVolumeInfo.getUpdatedAt()).isNotNull();
 
     // List volumes
     System.out.println("Testing list managed volumes..");
     Iterable<VolumeInfo> volumeInfosManaged =
         volumeOperations.listVolumes(CATALOG_NAME, SCHEMA_NAME);
-    assertEquals(1, getSize(volumeInfosManaged));
-    assertTrue(
-        contains(
-            volumeInfosManaged,
-            managedVolumeInfo,
-            (volume) -> {
-              assertThat(volume.getName()).isNotNull();
-              return volume.getName().equals(VOLUME_NAME);
-            }));
+    assertThat(getSize(volumeInfosManaged)).isEqualTo(1);
+    assertThat(volumeInfosManaged)
+        .as("Volume with name '%s' should exist", VOLUME_NAME)
+        .anySatisfy(volume -> assertThat(volume.getName()).isNotNull().isEqualTo(VOLUME_NAME));
 
     // NOW Update the schema name
     schemaOperations.updateSchema(
         SCHEMA_FULL_NAME, new UpdateSchema().newName(SCHEMA_NEW_NAME).comment(SCHEMA_COMMENT));
     // get volume
     VolumeInfo volumePostSchemaNameChange =
         volumeOperations.getVolume(CATALOG_NAME + "." + SCHEMA_NEW_NAME + "." + VOLUME_NAME);
-    assertEquals(volumePostSchemaNameChange.getVolumeId(), managedVolumeInfo.getVolumeId());
+    assertThat(managedVolumeInfo.getVolumeId()).isEqualTo(volumePostSchemaNameChange.getVolumeId());
 
     // test delete parent schema when volume exists
     assertThrows(
```

Comment:
we also want to replace `assertThrows` from JUnit with `assertThatThrownBy(() -> ...).isInstanceOf(Exception.class).hasMessage("...")`

Reply:
Done: [749854a](https://github.com/unitycatalog/unitycatalog/pull/279/commits/749854ad8ff979537e9bfaf3cdad93340495809f)